### DISCUSSION TITLE
Where is python 3.10.7?

### DISCUSSION THREAD

**Discussion by ColemanDunn** [pyenv/pyenv#2478] [Q&A] [answered]
When I execute `pyenv install -l`, it does not list 3.10.7 which was released over a month ago. 3.11.0rc2 is also missing. How long does it take for new releases to show up?

**Comment by native-api** [comment, marked as answer]
https://github.com/pyenv/pyenv/releases/tag/v2.3.5

We release about once a month. In the meantime, you can install a head version of Pyenv.
